README.md

```diff
@@ -1 +1,17 @@
-# python-swmrtools
+# swmr_tools
+
+swmr_tools is a Python library developed by Diamond Light Source ltd designed to facilitate reading and processing data from nexus (hdf5) files as they are being written
```

Comment:
Capitalise `Ltd`, also NeXus

Reply:
python-swmrtools won't be an importable name (I think ... the interpreter will view it as an expression to be evaluated). Can change the module name to python_swmrtools and that should work fine.